### ISSUE TITLE
formatter flattens contents of <script> tags

### ISSUE THREAD

**Issue by simonmcconnell** [surface-ui/surface_formatter#27]
it turned my <script> tag into this:

``` html
<script>
  document.addEventListener('alpine:init', () => {
  console.log('apline inited');
  Alpine.data('columnFilterOptions', () => ({
  all: true,
  open: false,
  options: [],
  initialSelection: [],
  selected: [],
  available: [],
  selectAll() {
  this.selected = this.options;
  },
  selectNone() {
  this.selected = [];
  },
  clear() {
  this.open = false;
  this.selected = this.initialSelection = this.options;
  },
  cancel() {
  this.open = false;
  this.selected = this.initialSelection;
  },
  apply() {
  this.open = false;
  this.initialSelection = this.selected;
  },
  init() {
  console.log('col filt init')
  this.$nextTick(() => {
  this.$watch('selected', (value) => {
  if (this.options.every((x) => { return this.selected.includes(x) })) {
  this.all = true;
  } else {
  this.all = false;
  }
  })
  })
  }
  }))
  })
</script>
```

Should it ignore contents of script tags or use a JS formatter?

**Comment by paulstatezny** [comment]
@simonmcconnell Thanks for reporting this! Sorry for the delay in responding.

I think the proper starting point would be to have the formatter simply leave the contents of script tags alone￼￼￼. I will respond back when this is patched.￼

**Comment by paulstatezny** [comment]
@simonmcconnell I just realized I never responded back to this. My apologies! This was patched in v0.5.1 in early July.

**Comment by simonmcconnell** [comment]
No problemo! I was aware because I had another issue open that you fixed in the same patch. Thanks for your contributions to planet earth 🌎.
